src/controllers/tenants.cr

```diff
@@ -39,6 +39,22 @@ class Tenants < Application
     Tenant.find!(params["id"].to_i64).delete
   end
 
+  get "/current_limits", :current_limits do
```

Comment:
@tassja how can I expose this route to all logged-in users? (https://github.com/PlaceOS/staff-api/issues/161)

Reply:
Is that route not exposed to all users? There isn't any auth or log in checks? @stakach

Reply:
I would assume that the `before_action :admin_only` applies to all routes in the controller, so I guess my question is, how do I exempt the route from that.

Reply:
Oh sorry missed that

Reply:
`before_action :admin_only, except: :current_limits`

Reply:
Thanks